TTHAnalysis/cfg/run_susyMultilepton_cfg.py

```diff
@@ -210,6 +216,20 @@
     susyMultilepton_collections.update({ # for conversion studies
             "selectedPhotons"    : NTupleCollection("PhoGood", photonTypeSusy, 10, help="Selected photons"),
             }) 
+    del susyMultilepton_collections["discardedJets"]
```

Comment:
I see that the modification to the cleaning of jets with leptons is in #143. Should be ok when both are merged, to be verified.